Mixpanel/MPUIControlBinding.m

```diff
@@ -177,7 +177,9 @@ - (void)stop
         // remove target-action pairs
         for (UIControl *control in self.appliedTo.allObjects) {
             if (control && [control isKindOfClass:[UIControl class]]) {
-                [self stopOnView:control];
+                dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Can you use `NSThread mp_safelyRunOnMainThreadSync:`? From https://github.com/mixpanel/mixpanel-iphone/blob/master/Mixpanel/NSThread+MPHelpers.m#L13. I've been trying to make that the standard way to do this in the SDK. Thanks!

Reply:
Sure, I changed it as you proposed.